Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/AppUpdaterCommandFactory.cs

```diff
@@ -119,13 +120,30 @@ public ICheckDiskSpace CreateCheckDiskSpaceCommandForDiff(int versionId, AppUpda
             }
 
             AppDiffSummary diffSummary = context.App.RemoteMetaData.GetDiffSummary(versionId, cancellationToken);
-            return new CheckDiskSpaceCommand(diffSummary, context.App.LocalDirectory.Path, biggestFileSize);
+            
+            var appDownloadDirectory = context.App.DownloadDirectory;
+            var destinationFilePath = appDownloadDirectory.GetDiffPackagePath(versionId);
+            sizeProcessedFiles += FileOperations.GetSizeFile(destinationFilePath);
+            
+            string destinationMetaPath = appDownloadDirectory.GetDiffPackageMetaPath(versionId);
+            sizeProcessedFiles += FileOperations.GetSizeFile(destinationMetaPath);
+            
+            return new CheckDiskSpaceCommand(diffSummary, context.App.LocalDirectory.Path, biggestFileSize, sizeProcessedFiles);
         }
 
         public ICheckDiskSpace CreateCheckDiskSpaceCommandForContent(int versionId, AppUpdaterContext context, CancellationToken cancellationToken)
         {
             AppContentSummary contentSummary = context.App.RemoteMetaData.GetContentSummary(versionId, cancellationToken);
-            return new CheckDiskSpaceCommand(contentSummary, context.App.LocalDirectory.Path);
+
+            long sizeProcessedFiles = 0;
+            var appDownloadDirectory = context.App.DownloadDirectory;
+            var destinationFilePath = appDownloadDirectory.GetContentPackagePath(versionId);
```

Comment:
do not use `var` is resulting type is unclear

Reply:
done